tests/queries/0_stateless/01470_columns_transformers2.sql

```diff
@@ -0,0 +1,7 @@
+DROP TABLE IF EXISTS columns_transformers;
+
+CREATE TABLE columns_transformers (i int, j int, k int, a_bytes int, b_bytes int, c_bytes int) Engine=TinyLog;
+INSERT INTO columns_transformers VALUES (100, 10, 324, 120, 8, 23);
+SELECT  * EXCEPT 'bytes', COLUMNS('bytes') APPLY formatReadableSize FROM columns_transformers;
```

Comment:
It looks inconsistent as COLUMNS require parens for regexp and EXCEPT does not.
Does it work with parens?

Reply:
> Does it work with parens?

It does. We can leave out parens when there is only one argument for column transformers